integreat_cms/api/v3/locations.py

```diff
@@ -98,6 +98,7 @@ def transform_poi_translation(poi_translation):
             "slug": poi.organization.slug,
             "name": poi.organization.name,
             "logo": poi.organization.icon.url if poi.organization.icon else None,
+            "website": poi.organization.website if poi.organization.website else None,
```

Comment:
```suggestion
            "website": poi.organization.website,
```
If we don't want `None` values there in the long time, I guess it's fine to go with an empty string for the time being. This also helps the apps if they can be sure that the result is a string and not a string or `None`.